poetry.lock

```diff
@@ -1,18 +1,26 @@
+# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
```

Comment:
Sadly, we are using an ancient poetry, 1.2.2...

Reply:
The pipelines use 1.4 and our internal builds are finally upgraded to 1.8. I'll post a PR to update Poetry.